specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -1850,6 +1850,41 @@
         "encryptedCredential": {
           "type": "string",
           "description": "The encrypted credential used for authentication. Credentials are encrypted using the integration runtime credential manager. Type: string."
+        },
+        "allowZeroDateTime": {
```

Comment:
Please update examples to reflect these properties and how are they used.

Reply:
We have generic examples, do not need to write examples per connector